try:
    a = 3
    b = 6
    print(a + b)
except Exception as e:
    print(e)
